tests/queries/0_stateless/01247_optimize_distributed_group_by_sharding_key_dist_on_dist.sql

```diff
@@ -21,20 +21,18 @@ set optimize_skip_unused_shards=1;
 select 'Distributed(number)-over-Distributed(number)';
 create table dist_layer_01247 as data_01247 engine=Distributed(test_cluster_two_shards, currentDatabase(), data_01247, number);
 create table dist_01247 as data_01247 engine=Distributed(test_cluster_two_shards, currentDatabase(), dist_layer_01247, number);
-select count(), * from dist_01247 group by number;
+select count(), * from dist_01247 group by number order by number limit 1;
 drop table if exists dist_01247;
 drop table if exists dist_layer_01247;
 
 select 'Distributed(rand)-over-Distributed(number)';
 create table dist_layer_01247 as data_01247 engine=Distributed(test_cluster_two_shards, currentDatabase(), data_01247, number);
 create table dist_01247 as data_01247 engine=Distributed(test_cluster_two_shards, currentDatabase(), dist_layer_01247, rand());
-select count(), * from dist_01247 group by number;
+select count(), * from dist_01247 group by number order by number limit 1;
 drop table if exists dist_01247;
 drop table if exists dist_layer_01247;
 
 select 'Distributed(rand)-over-Distributed(rand)';
 create table dist_layer_01247 as data_01247 engine=Distributed(test_cluster_two_shards, currentDatabase(), data_01247, rand());
 create table dist_01247 as data_01247 engine=Distributed(test_cluster_two_shards, currentDatabase(), dist_layer_01247, number);
-select count(), * from dist_01247 group by number;
-drop table dist_01247;
```

Comment:
Don't understand this change, will return DROP queries...

Reply:
When test DROP TABLE at the end you cannot attach with the client and reproduce the failure, that's why I'm trying to keep them.
@alexey-milovidov Is there some unspoken rule that said that tables should be removed after the test?

Reply:
It's just for convenience when you run .sql files manually and don't want leftovers.